src/transform/__tests__/transform-utils-test.js

```diff
@@ -43,4 +43,75 @@ describe('transform-utils', () => {
     expect(disjunction).toEqual(expected);
   });
 
+  it('increaseQuantifierByOne', () => {
+    const quantifiers = [{
+        type: 'Quantifier',
+        kind: '*',
+        greedy: true
```

Comment:
BTW, I added recently ESLint checks, you'll need to do `npm install` again to get the latest changes. There should be a pre-commit hook which runs both tests, and ESLint. You can also run it manually as `npm run eslint` in case. Otherwise, publish was blocked by it (https://github.com/DmitrySoshnikov/regexp-tree/commit/4d2bbeeee2414f99702cdeb33eb462cb8ddf19ab).

Reply:
Awesome! I managed to configure the hooks locally and it works like a charm. My PRs should be ok in the future ;)

Reply:
No worries, thanks!